core/src/global.ts

```diff
@@ -1,18 +1,8 @@
+import type { WindowCapacitor } from './definitions-internal';
 import { legacyRegisterWebPlugin } from './legacy/legacy-web-plugin-merge';
-import { initCapacitorGlobal } from './runtime';
 import type { WebPlugin } from './web-plugin';
 
-export const Capacitor = /*#__PURE__*/ initCapacitorGlobal(
-  (typeof globalThis !== 'undefined'
-    ? globalThis
-    : typeof self !== 'undefined'
-    ? self
-    : typeof window !== 'undefined'
-    ? window
-    : typeof global !== 'undefined'
-    ? global
-    : {}) as any,
-);
+export const Capacitor = (window as WindowCapacitor).Capacitor;
```

Comment:
Avoid using `window` directly like this because it'll likely fail in many SSR and node testing framework implementations. In today's landscape, and the minimum of what we require for Capacitor, it might be safe to start using just `globalThis` instead. Otherwise you can do this check:

```
(typeof globalThis !== 'undefined'
    ? globalThis
    : typeof self !== 'undefined'
    ? self
    : typeof window !== 'undefined'
    ? window
    : typeof global !== 'undefined'
    ? global
    : {}) as any,
```

Reply:
Fixed it and also added a fallback